custom/enikshay/two_b_release_1/management/commands/enikshay_2b_case_properties.py

```diff
@@ -239,8 +238,7 @@ def migrate_person(self, person, occurrences, episodes):
         if occurrence:
             episodes = [case for case in episodes
                         if not case.closed and case.type == CASE_TYPE_EPISODE
-                        and any([index.referenced_id == occurrence.case_id for index in case.indices])
-                        and case.dynamic_case_properties().get('episode_type') == "confirmed_tb"]
+                        and any([index.referenced_id == occurrence.case_id for index in case.indices])]
```

Comment:
nit: I think `any` works a little faster with a generator since it doesn't compute all the values.